sos/plugins/docker.py

```diff
@@ -25,37 +25,52 @@ class Docker(Plugin):
     profiles = ('virt',)
     docker_bin = "docker"
 
-    option_list = [("all", "capture all container logs even the "
-                    "terminated ones", 'fast', False)]
+    option_list = [
+        ("all", "enable capture for all containers, even containers "
+            "that have terminated", 'fast', False),
+        ("logs", "capture logs for running containers",
+            'fast', False)
+    ]
 
     def setup(self):
         self.add_copy_spec([
             "/var/lib/docker/repositories-*"
         ])
 
-        self.add_cmd_output([
-            "journalctl -u docker",
-            "{0} info".format(self.docker_bin),
-            "{0} ps".format(self.docker_bin),
-            "{0} images".format(self.docker_bin)
-        ])
+        for cmd in ['info', 'ps', 'ps -a', 'images', 'version']:
```

Comment:
Should inspect be part of this list? Do we need to have an all or running option like with logs?